web/src/agents/node/get_balance.rs

```diff
@@ -0,0 +1,41 @@
+use at2_node::client::{self, Client};
+use at2_ns::ThinUser;
+use wasm_bindgen_futures::spawn_local;
+use yew::worker::*;
+
+use crate::config::Config;
+
+/// Get the balance of an user
+pub struct GetBalance {
+    link: AgentLink<Self>,
+    client: Client,
+}
+
+impl Agent for GetBalance {
+    type Reach = Context<Self>;
+    type Message = (HandlerId, Result<u64, client::Error>);
+    type Input = ThinUser;
+    type Output = Result<u64, client::Error>;
+
+    fn create(link: AgentLink<Self>) -> Self {
+        let conf = Config::parse().unwrap(); // TODO unwrap
```

Comment:
Resolve TODOs, also on line 25.

Reply:
Most TODO resolved and `unwrap` handled.